python/dgl/transforms/functional.py

```diff
@@ -70,7 +70,9 @@
     'adj_product_graph',
     'adj_sum_graph',
     'reorder_graph',
-    'norm_by_dst'
+    'norm_by_dst',
+    'rwpe',
+    'laplacian_pe'
```

Comment:
Perhaps we can rename `laplacian_pe` to `lappe` for consistency, or from `rwpe` to `rw_pe`.

Reply:
done